styles/pov.h

```diff
@@ -4,38 +4,40 @@
 // TODO(hubbe): Make it work with FastLED
 #if defined(ENABLE_WS2811)
 
-/* 
+
+// Usage: StylePOV<>
```

Comment:
I think you made it more confusing since StylePOV and ContinousPOV now have been moved away from their descriptions.

Reply:
Restored